api/src/api/server.clj

```diff
@@ -29,6 +31,15 @@
 
 (defn health [_] {:status 200, :body "Ok"})
 
+(defn status [_]
+  (let [main-deps {:gf (gf-service/ping)}
+        other-deps {}
```

Comment:
should we add "datomic-transactor" to other deps?

Reply:
Lets add other deps as separate, because for others we need to check ENV variables and decide if need to check for those